app/experimenter/nimbus-ui/src/lib/visualization/utils.tsx

```diff
@@ -10,6 +10,9 @@ import { AnalysisData } from "./types";
 export const analysisAvailable = (analysis: AnalysisData | undefined) =>
   analysis?.show_analysis === true && analysis?.overall !== null;
 
+export const analysisUnavailable = (analysis: AnalysisData | undefined) =>
```

Comment:
To be clear, this is what fixes the bug. We have to check for `analysis` truthiness.